app/participants/admit.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from app.participants.knight import Knight
+
+
+def create_knights(knight_config: dict) -> Knight:

```

Comment:
You create a single knight here, so you should name the function as `create_knight`.